packages/help-center/src/hooks/use-content-filter.ts

```diff
@@ -95,6 +95,33 @@ export const useContentFilter = ( node: HTMLDivElement | null ) => {
 					}
 				},
 			},
+			/**
+			 * Add external link icons to links that open in new tabs.
+			 */
+			{
+				pattern: 'a[target="_blank"]',
```

Comment:
Apparently, support links also have target="_blank", we should use `rel="noopener"` instead. You can see an arrow was added to every link. I found an external link in "Connect a GoDaddy Domain" article under `Option 1: Simple Setup`, there is a [Domain Connect standard](https://www.domainconnect.org/) link.

Also, I think the arrow needs some margin or white space before it.